src/components/Hoverable/index.js

```diff
@@ -42,10 +42,6 @@ class Hoverable extends Component {
                 if (!scrolling && this.isHoveredRef) {
                     this.setState({isHovered: this.isHoveredRef}, this.props.onHoverIn);
                 } else if (scrolling && this.isHoveredRef) {
-                    /**
-                     * If the user has started scrolling and the isHoveredRef is true, then we should set the hover state to false.
-                     * This is to hide the existing hover and reaction bar.
-                     */
```

Comment:
you mean just the comments or the comment and the initial code `this.isHoveredRef = false;`

Reply:
Just the comments. Why we are deleting those comments?

Reply:
comm

> Just the comments. Why we are deleting those comments?

comments were related to this `this.isHoveredRef = false;`. So I removed them since I was taking off that piece of code.

Reply:
In the comment we are explaining why we are setting the hover state to false and we are still setting it to false.

Reply:
ok, I will revert.